test/docker/Dockerfile.cassandra

```diff
@@ -0,0 +1,12 @@
+ARG CASSANDRA_VERSION=latest
+
+FROM cassandra:$CASSANDRA_VERSION
```

Comment:
I was happy to know that having an argument in the `FROM` works fine, which means we can have this clean way of spinning up different Cassandra version while still using the original Cassandra Docker images.